meson.build

```diff
@@ -8,8 +8,11 @@ if host_machine.system() == 'managarm'
 			add_project_arguments('-fsized-deallocation', language: 'cpp')
 		endif
 
+		frigg_dep = dependency('frigg', method: 'pkg-config', allow_fallback: false)
```

Comment:
Why do we force the use of `pkg-config` and use `allow_fallback: false` here?

Reply:
If we want to allow fallback, then we need to provide a `frigg.wrap` inside `subprojects/` right?

I'll get rid of the required method I guess

Reply:
We might want to do that in the future. Even if we don't, this option is not really necessary, is it? If there is no fallback, it also won't be used.

Reply:
Not totally sure why, but if I simply omit `allow_fallback: false`, it errors. But adding `fallback: ['frigg', 'frigg_dep']` makes it work again